service/history/replicatorQueueProcessor.go

```diff
@@ -728,7 +728,7 @@ func (p *replicatorQueueProcessorImpl) getVersionHistoryItems(
 		return nil, nil, err
 	}
 
-	versionHistory, err := versionhistory.GetVersionHistory(versionHistories, versionHistoryIndex)
+	versionHistory, err := versionhistory.Get(versionHistories, versionHistoryIndex)
```

Comment:
i would prefer keeping the current function names

Reply:
> The importer of a package will use the name to refer to its contents

this package contains 3 structs, simple `Get` is confusing.
separating into 3 packages will also cause troubles: `versionhistory` vs `versionhistories` -> hard to read